VotingApplication/VotingApplication.Web/Api/Controllers/PollChoiceController.cs

```diff
@@ -56,6 +57,8 @@ public void Post(Guid pollId, ChoiceCreationRequestModel choiceCreationRequest)
                 poll.LastUpdatedUtc = DateTime.UtcNow;
 
                 context.SaveChanges();
+
+                ClientSignaller.SignalUpdate(poll.UUID.ToString());
```

Comment:
Does `.ToString()` want to have an explicit format string here, as this is being used as an identifier.